### DISCUSSION TITLE
Drag and drop text

### DISCUSSION THREAD

**Discussion by PascalLadalle** [4ian/GDevelop#3651] [Enhancement ideas (and their technical discussions)]
Not sure if it's post beta 89 or not, but I'm unable to drag and drop text in text boxes, it's quite inconvenient.
I can highlight text and drag it, but there is no cursor to allow me to drop it.
![image](https://user-images.githubusercontent.com/25978830/80521317-20aeaf80-898b-11ea-87dc-25cab9d970e4.png)

GD 5 beta 92 installed on Windows 10 64-bit

**Comment by 4ian** [comment]
Interesting, I'm never drag'n'dropping text ^^ Are you doing that often instead of copy-pasting?

Please also indicate if it's the web-app or not and your operating system, as this is highly related to your system.

**Comment by PascalLadalle** [comment]
Often, I don't know... often enough to make me report it. 😊 
It's because my current mouse is on the left, so copy-paste and mouse share the left hand. 
I prefer drag-n-dropping over moving my left hand to the keyboard or doing a clumsy right-handed copy-paste with my pinky. 🙂 

(report updated with details)
